### PR TITLE
feat: support testng 7.9.0

### PR DESCRIPTION
**Pull request by missedone**
https://github.com/testng-team/testng/pull/2932#issuecomment-1667264522

### CODE REVIEW

**Comment by missedone** [comment]
wait for testng 7.9.0 release, then will run the CI test again

**Comment by missedone** [comment]
@juherr  @krmahadevan pls review

File: remote7_9/pom.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <test.resources.dir>${basedir}/../remote/target/test-classes</test.resources.dir>
```
Comment (krmahadevan) [inline remote7_9/pom.xml:47]: This is just question out of curiosity. Are we intentionally running the tests that are located in https://github.com/testng-team/testng-remote/tree/master/remote or was this a copy paste discrepancy ?

Comment (missedone) [reply remote7_9/pom.xml:47]: It uses the same test suite xml to verify all the version of testng

**Review by krmahadevan — APPROVED** [review]
Since I am not entirely familiar with this codebase, it would be good if @juherr also helps take a look at this changes.

**Review by juherr — APPROVED** [review]
The change is good but does it mean that 7.9 is not backward compatible?

**Comment by missedone** [comment]
> The change is good but does it mean that 7.9 is not backward compatible?

Yes, not sure if it breaks anything else, but testng-remote needs update see https://github.com/testng-team/testng/pull/2932#issuecomment-1667264522

**Pull request merged**